cache.py

```diff
@@ -87,9 +88,7 @@ def keyFromArgs(f, userSensitive, languageSensitive, evaluatedArgs, path, args,
 	try:
 		appVersion = currentRequest.get().request.environ["CURRENT_VERSION_ID"].split('.')[0]
 	except:
-		appVersion = ""
-		# FIXME
-		#logging.error("Could not determine the current application id! Caching might produce unexpected results!")
+		appVersion = utils.projectID
```

Comment:
This fix is invalid. appVersion is supposed to contain the version (like ts-dev) - not the projectID. Reason is to prevent leaking from development versions into the live site

Reply:
what is your proposal then to fix this issue? `utils.projectID` looks for me as a fallback.

Reply:
When we cannot determine the version it doesn't matter which constant we would use as fallback.
Therefore I let the empty string in #54. Maybe we will find a better way to get the version or have to differentiate btw. local devserver and cloud ...